core/trino-main/src/test/java/io/trino/block/AbstractTestBlock.java

```diff
@@ -204,17 +204,17 @@ private void assertBlockSize(Block block)
     {
         // Asserting on `block` is not very effective because most blocks passed to this method is compact.
         // Therefore, we split the `block` into two and assert again.
-        long expectedBlockSize = copyBlockViaBlockSerde(block).getSizeInBytes();
+        long expectedBlockSize = copyBlockViaCopyRegion(block).getSizeInBytes();
```

Comment:
is copying method change relevant?

Reply:
> is copying method change relevant?

Yes as if you encode/decode block it might have different size now (different block type)